.env.example

```
@@ -1,10 +1,19 @@
+NEXT_PUBLIC_SUPABASE_PUBLISHABLE_KEY=supabase_publishable_key
+
+# Clerk auth configuration
 CLERK_SECRET_KEY=clerk_secret_key
 CLERK_DOMAIN=clerk_domain
-OPENAI_API_KEY=your_openai_api_key
-NEXT_PUBLIC_POSTHOG_KEY=posthog_key
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=clerk_publishable_key
+
+# Sentry error tracking configuration
+SENTRY_DSN=sendry_dsn_url
 NEXT_PUBLIC_SENTRY_DSN=sendry_dsn_url
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SENTRY_DSN key should go before the SENTRY_DSN key

(UnorderedKey)